### PR TITLE
lsps2: cleanup expired promises

### PR DESCRIPTION
**Pull request by JssDWt**
Fixes https://github.com/breez/lspd/issues/121

### CODE REVIEW

File: postgresql/migrations/000014_lsps2_buy.up.sql [hunk 1]
```diff
@@ -34,5 +34,7 @@ CREATE INDEX idx_lsps2_bought_channels_registration_id ON lsps2.bought_channels
 
 CREATE TABLE lsps2.promises (
 	promise varchar PRIMARY KEY,
-	token varchar NOT NULL
-)
+	token varchar NOT NULL,
+	valid_until varchar NOT NULL
```
Comment (roeierez) [inline postgresql/migrations/000014_lsps2_buy.up.sql:38]: Doesn't adding a not null column require default value?

Comment (JssDWt) [reply postgresql/migrations/000014_lsps2_buy.up.sql:38]: It's always set here:
https://github.com/breez/lspd/pull/130/files#diff-e721fefad0f3dcd1d7884161d3886c91ff99412d1d2cff025e99df0384c7cce8R244

Comment (roeierez) [reply postgresql/migrations/000014_lsps2_buy.up.sql:38]: Ah.. makes sense

**Comment by JssDWt** [comment]
This PR was pointing to the wrong branch, but it was merged here: https://github.com/breez/lspd/commit/084a4d8d385b8ec0d835c4802d0707f463d5f74f

**Pull request closed without merging**